app/components/Layout/Header.jsx

```diff
@@ -214,7 +214,7 @@ class Header extends React.Component {
                         <li className={name === account_display_name ? "current-account" : ""} key={name}>
                             <a href onClick={this._accountClickHandler.bind(this, name)}>
                                 <div className="table-cell"><AccountImage style={{position: "relative", top: 5}} size={{height: 20, width: 20}} account={name}/></div>
-                                <div className="table-cell" style={{paddingLeft: 10}}><span>{name}</span></div>
+                                <div className="table-cell" style={{paddingLeft: 10, verticalAlign: "middle"}}><div className="inline-block"><span className="lower-case">{name}</span></div></div>
```

Comment:
Your new div and verticalAlign style causes alignment issues, change it so the only change is setting `className="lower-case"` on the span and it looks fine.

![image](https://user-images.githubusercontent.com/6890015/31654207-9338be40-b325-11e7-8a9f-ab82c5f416e7.png)